### ISSUE TITLE
No capability for global reduction only called after remeshing

### ISSUE THREAD

**Issue by pgrete** [parthenon-hpc-lab/parthenon#591]
We have the need to call a global reduction after mesh refinement, specifically, we are interested in the minimum `dx` (though in principle it could be recovered from parsing the entire mesh structure on each rank).
That raised the point whether this is a capability that'd also be of interested in other codes.
This would be similar to the feature we reenabled recently for "one off" call after mesh refinement with the difference to allow for reduction in that call.
Is this something you'd think to be useful to have, , @Yurlungur @jdolence @brryan ?

Cc @forrestglines

**Comment by brryan** [comment]
Very generally I'm in favor of exposing global reductions as an option throughout the code, because this isn't something that can easily be worked around by downstream developers. I *think* we (phoebus team) could make use of global reductions after mesh refinement, for example if we wanted to evaluate the average number of particles per zone after refinement.

**Comment by Yurlungur** [comment]
I agree with @brryan . There's no obvious use-case that comes to my mind right now. But I think this is a good capability to have.
